tests/test_common.c

```diff
@@ -169,7 +170,7 @@ void setup_test_store_context(signal_protocol_store_context **context, signal_co
 
     signal_protocol_store_context *store_context = 0;
     result = signal_protocol_store_context_create(&store_context, global_context);
-    ck_assert_int_eq(result, 0);
+    assert(result == 0);
```

Comment:
Swapped to assert in this one case so that this setup function can be used by the fuzzer which is not running as part of a check test suite.